services/movie_session.py

```diff
@@ -0,0 +1,65 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: str
+) -> MovieSession:
+    if cinema_hall_id:
+        try:
+            cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+        except CinemaHall.DoesNotExist:
+            raise ValueError(f"CinemaHall with"
+                             f" id={cinema_hall_id} does not exist.")
+    if movie_id:
+        try:
+            movie = Movie.objects.get(id=movie_id)
+        except Movie.DoesNotExist:
+            raise ValueError(f"CinemaHall with id={movie_id} does not exist.")
+
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=cinema_hall,
+        movie=movie
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet[MovieSession]:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(
+        movie_session_id: str = None
+) -> MovieSession:
+    if movie_session_id:
+        try:
+            return MovieSession.objects.get(id=movie_session_id)
+        except MovieSession.DoesNotExist:
+            raise ValueError(f"MovieSession with "
+                             f"id={movie_session_id} does not exist.")
+    return MovieSession.objects.all()

```

Comment:
Returning `MovieSession.objects.all()` when `movie_session_id` is not provided is not appropriate for a function named `get_movie_session_by_id`. Consider raising an exception or returning `None` if `movie_session_id` is not provided.